paper.tex

```diff
@@ -63,7 +63,7 @@ \subsection{Participants}
 There are four major roles in the decentralized data marketplace (Figure \ref{fig:system_design}).
 
 \subsubsection{Registrar}
-Registrars have access to create and maintain Registration Contracts, a lookup table of participants in decentralized data marketplace. 
+Registrars have access to create and maintain registration contracts, a lookup table of participants in decentralized data marketplace. 
```

Comment:
I would like to keep the upper case form " Registration Contract".

Reply:
What about "product contract" and "purchase contract"?

Reply:
I defer to @twzjwang .